frontend/src/pages/Player/PlayerHook/PlayerHook.tsx

```diff
@@ -315,6 +314,43 @@ export const usePlayer = (): ReplayerContextInterface => {
 		[getChunkIdx, showPlayerMouseTail, state.sessionMetadata.startTime],
 	)
 
+	const loadEventChunk = useCallback(
+		async (_i: number) => {
+			try {
+				const url = await fetchEventChunkURL({
```

Comment:
Not blocking since pre-existing- noticed above that `fetchEventChunkURL` uses `rawFetchEventChunkURL` which hardcodes `skip: true` - just wanted to make sure this was expected

Saw that we cleaned up a lot of calls to `fetchEventChunkURL`

Reply:
this is expected - that `skip` means we never call the query when we actually instantiate the hook `useGetEventChunkUrlQuery`. that hook just exposes a `{ refetch: rawFetchEventChunkURL }` refetch fn that we later use, and calling the refetch method makes the query.